tools/targets/STM32_gen_PeripheralPins.py

```diff
@@ -0,0 +1,1244 @@
+"""
+ *******************************************************************************
+ * Copyright (c) 2018, STMicroelectronics
```

Comment:
Quick question , as these are mbed implementation files (also what it generates), why not Apache license?